### ISSUE TITLE
Regarding the fix for teleporting inside dungeon while in combat

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5629]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
the report is here https://github.com/dalaranwow/dalaran-wow/issues/5619 and here https://github.com/dalaranwow/dalaran-wow/issues/5470
I just wanted to say that last weekend a gm answered my ticket and told me that monday it will get fixed when fixes are going to be deployed. I checked monday evening if the bug is still present and i noticed there was a fix in but it still doesnt work completly. You can still enter dungeon while in combat but only if you stop attacking your target, for example if you attack someone - you get in combat, then you cant enter the dungeon but if you stop attacking him while you're still in combat then you can enter. Also remiding that you can accept the battleground invite even if in combat that's also an issue, ppl can que bg just to avoid dying in stormwind/orgrimmar.
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by MalkaviaDW** [comment]
Hello, thanks for the report it will be fixed with the next maintenance. 

Cheers!
